src/components/HexGrade/HexGrade.stories.tsx

```diff
@@ -33,6 +33,9 @@ playground.argTypes = {
   isInversed: {
     control: 'boolean',
   },
+  margin: {
+    control: 'object',
```

Comment:
I usually disable controls for `margin` since it's not very user-friendly to edit this in Storybook

Reply:
Agree it's not ideal but no one forces users to edit that - I think it's useful at least for me!